test/expect/TestBatched.test_numToTensor.expect

```diff
@@ -0,0 +1,28 @@
+graph(%a.1_data : Dynamic
+      %a.1_mask : Dynamic
+      %a.1_dims : Dynamic) {
+  %3 : int = prim::Constant[value={1}]()
```

Comment:
Not an issue in this diff, but there's a lot of constant index manipulation happening here. I wonder if after https://github.com/pytorch/pytorch/pull/8808/files lands we should add a call to constant prop at the end of the batching pass to clean it up